src/Bundle/Grid/Parser/OptionsParser.php

```diff
@@ -29,13 +29,16 @@ final class OptionsParser implements OptionsParserInterface
     public function __construct(
         ContainerInterface $container,
         ExpressionLanguage $expression,
-        PropertyAccessorInterface $propertyAccessor
+        PropertyAccessorInterface $propertyAccessor,
     ) {
         $this->container = $container;
         $this->expression = $expression;
         $this->propertyAccessor = $propertyAccessor;
     }
 
+    /**
+     * @param null $data
```

Comment:
```suggestion
     * @param mixed $data
```
or even better to list explicitly possible types